tinygrad/device.py

```diff
@@ -278,6 +278,14 @@ def get_linearizer(self, *ast:LazyOp) -> Linearizer:
         timed = sorted([(nm, tk, time_linearizer(tk, test_rawbuffers, allow_test_size=False, clear_l2=True)) for nm, tk in lins], key=lambda x: x[2])
         if DEBUG >= 1: print("  <  ".join(f"{nm:6s} : {lin.colored_shape(30, dense=True)} : {tm*1e6:8.2f} us" for nm, lin, tm in timed))
         k = timed[0][1]
+
+        if getenv("BEAM_VERIFY", 0): # compare the beam generated values with required_opts only
+          from test.external.fuzz_linearizer import compare_linearizer
```

Comment:
I think core tinygrad should not import from test, even if it's a debug feature. That's why I proposed logging in core and use fuzzer to read the file.